.github/workflows/ci.yml

```diff
@@ -58,6 +58,18 @@ jobs:
   Test_pyramid-app:
     name: Test pyramid-app
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:15.3-alpine
+        ports:
+        - 5439:5432
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
This is the cookiecutter's own CI. This CI job generates a new project using the `pyramid-app` cookiecutter and runs the new project's `make sure` command. It now needs to have a DB running on CI because commands run by `make sure` will now run `init_db.py` which'll try to connect to the DB.